testsuite/smoke-test/src/genesis.rs

```diff
@@ -10,20 +10,33 @@ use crate::{
 };
 use anyhow::anyhow;
 use aptos_config::config::NodeConfig;
+use aptos_logger::prelude::*;
 use aptos_temppath::TempPath;
 use aptos_types::{transaction::Transaction, waypoint::Waypoint};
-use forge::{get_highest_synced_version, LocalNode, Node, NodeExt, SwarmExt};
+use forge::{get_highest_synced_version, LocalNode, Node, NodeExt, SwarmExt, Validator};
 use move_core_types::language_storage::CORE_CODE_ADDRESS;
 use regex::Regex;
+use std::time::Instant;
 use std::{fs, process::Command, str::FromStr, time::Duration};
 
-fn update_node_config_restart(validator: &mut LocalNode, mut config: NodeConfig) {
+async fn update_node_config_restart(validator: &mut LocalNode, mut config: NodeConfig) {
```

Comment:
I'm not against this, but is there a specific reason?

Reply:
Ah yeah, I'll make it non-async. I did some await's in the method in a previous iteration.